classes/Hook.php

```diff
@@ -1004,7 +1014,16 @@ public static function coreRenderWidget($module, $hook_name, $params)
             return null;
         }
 
-        return $module->renderWidget($hook_name, $params);
+        try {
+            return $module->renderWidget($hook_name, $params);
+        } catch (\Exception $e) {
```

Comment:
```suggestion
        } catch (Exception $e) {
```

there's no need to use `\` right?

Reply:
@lmeyer1 A small fix and we merge.